### PR TITLE
Allow deletion and stopping of vms in retired state

### PR DESCRIPTION
**Pull request by mtkh**

### CODE REVIEW

**Comment by mtkh** [comment]
ok, I will update the MR accordingly afterwards

**Comment by mtkh** [comment]
I have updated the parameters with new names

**Comment by mtkh** [comment]
retest this please

File: igvm/commands.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def vm_start(vm_hostname):
 @with_fabric_settings
 def vm_stop(vm_hostname, force=False):
     """Gracefully stop a VM"""
-    with _get_vm(vm_hostname) as vm:
+    with _get_vm(vm_hostname, allow_retired=True) as vm:
```
Comment (TuxPowered42) [inline igvm/commands.py]: A VM which is retired must already be stopped. No live VM can be retired.

**Pull request merged**